projects/packages/publicize/src/class-publicize-main.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Main Publicize class.
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Publicize;
+
+/**
+ * The class to configure and initialize the publicize package.
+ */
+class Publicize_Main {
+	/**
+	 * Contains an instance of class 'Publicize' which loads Keyring, sets up services, etc.
+	 *
+	 * @var Publicize Instance of Publicize
+	 */
+	public $publicize;
+
+	/**
+	 * To configure the publicize package, when called via the Config package.
+	 */
+	public static function configure() {
+		add_action( 'plugins_loaded', array( __CLASS__, 'on_plugins_loaded' ) );
+	}
+
+	/**
+	 * To configure the publicize package, when called via the Config package.
+	 */
+	public static function on_plugins_loaded() {
+		global $publicize_ui;
```

Comment:
Should we check the global `$publicize` variable here, or perhaps the current value of `$publicize_ui`? It might be worth checking that it hasn't already been initialised.